### PR TITLE
prql-query v0.0.14

### PR DESCRIPTION
**Pull request by pthariensflame**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by snth** [comment]
@pthariensflame thank you for this.

Apologies, I don't use homebrew so for me it's difficult supporting this so any help or pointers would be appreciated.

Does this mean that for every release of prql-query this needs to be updated? What is that SHA compiled from?

**Comment by pthariensflame** [comment]
@snth No worries!

The SHA is of the file specified in `url`, so it changes as the downloaded file does, naturally.  A change like this is needed every release of `prql-query` but it doesn't have to be manual; `prql-compiler`'s formula has been being updated by a GitHub Action, IIUC.

If you'd like, I can add to this PR by formatting and cleaning up the file to Homebrew's intended form.  ❤️

**Comment by snth** [comment]
Thank you @pthariensflame . That would be amazing!

**Comment by pthariensflame** [comment]
I cleaned up both formulae and made the GitHub Action enforce style from now on.

**Comment by snth** [comment]
Thank you very much @pthariensflame . Are you one of the homebrew maintainers/volunteers? Does this mean prql-query is now part of homebrew core like prql-query or it's still a tap/cask (apologies, I forgot the correct term and difference between the two)?

Also, would you say this is good to merge now?

I'm going to cut a prql-query release soon (today or tomorrow possibly) so we'll probably see soon if it all works.

**Comment by pthariensflame** [comment]
This is still a separate tap, not “official”!  And no, I’m not part of Homebrew, just a knowledgeable volunteer.  😄  Something failed here so I want to investigate that before merging, but it should be good to go soon!

**Comment by pthariensflame** [comment]
Okay; this should be good to merge as long as the tests pass!

**Comment by snth** [comment]
Great, thanks so much!

**Review by snth — APPROVED** [review]
LGTM Thank you

File: .github/workflows/tests.yaml
Comment (max-sixty) [inline .github/workflows/tests.yaml:20]: Great, thanks for changing this

File: Formula/prql-compiler.rb
Comment (max-sixty) [inline Formula/prql-compiler.rb:2]: Not a huge deal, but the existing line is a canonical sentence we use in lots of places; do we have a strong reason to change it? If so, we should change it everywhere...

File: Formula/prql-compiler.rb
Comment (max-sixty) [inline Formula/prql-compiler.rb:1]: Do you think we could change the name of this file to `prqlc.rb`? Ideally the recent change should mean we use `prql/prql/prqlc` rather than `prql/prql/prql-compiler` (and I don't know whether it's possible to deprecate the old one or we just remove it, fine to remove if it that's easier...)

**Review by max-sixty — COMMENTED** [review]
Thanks a lot for spotting this @pthariensflame and making the changes.

I left a couple of comments around the `prql-compiler` changes, let me know what you think.

File: Formula/prql-compiler.rb
Comment (pthariensflame) [reply Formula/prql-compiler.rb:2]: I only changed it to avoid hitting the length limit Homebrew prefers for descriptions.  Is there a better way to do that you'd prefer?

File: Formula/prql-compiler.rb
Comment (pthariensflame) [reply Formula/prql-compiler.rb:1]: It's entirely possible to both change the canonical formula name and keep the old name around as an alias!  I can make that change in a few hours when I get home.

File: Formula/prql-compiler.rb
Comment (max-sixty) [reply Formula/prql-compiler.rb:2]: Oh interesting! That's fine then. I guess we could shorten "language" to "lang" but I probably prefer your change

File: Formula/prql-compiler.rb
Comment (max-sixty) [reply Formula/prql-compiler.rb:1]: Super! Thank you

File: .github/workflows/tests.yaml
Comment (max-sixty) [reply .github/workflows/tests.yaml:20]: FYI I adjusted this in `main` to stop it breaking.

@snth please can you review the `prql-query` part of this so we can merge this PR?

File: Formula/prql-compiler.rb
Comment (pthariensflame) [reply Formula/prql-compiler.rb:1]: Sorry about so long a delay!  I'll make it happen in another PR.

File: .github/workflows/tests.yaml
Comment (max-sixty) [reply .github/workflows/tests.yaml:20]: Oh, I guess this was the approval? https://github.com/PRQL/homebrew-prql/pull/10#pullrequestreview-1262596083

Sorry, so maybe you were waiting on me!!

I'll try and merge main back in and then merge this. Sorry for the delay @pthariensflame .

Comment (pthariensflame) [reply .github/workflows/tests.yaml:20]: No need to worry!  I've been held up too.

**Comment by max-sixty** [comment]
Because of https://github.com/PRQL/homebrew-prql/issues/11, I'm going to merge and then we can iterate...

**Comment by snth** [comment]
Thank you

**Pull request merged**
